recipes/glib/all/conanfile.py

```diff
@@ -97,10 +100,17 @@ def generate(self):
 
         tc.project_options["selinux"] = "enabled" if self.options.get_safe("with_selinux") else "disabled"
         tc.project_options["libmount"] = "enabled" if self.options.get_safe("with_mount") else "disabled"
-        if self.settings.os == "FreeBSD":
+        if self.settings.os == "FreeBSD" or self.settings.os == "Neutrino":
             tc.project_options["xattr"] = "false"
         tc.project_options["tests"] = "false"
         tc.project_options["libelf"] = "enabled" if self.options.get_safe("with_elf") else "disabled"
+
+        if self.settings.os == "Neutrino":
+            tc.cross_build["host"]["system"] = "qnx"
+            tc.c_args = ["-Wno-nonnull", "-Wno-format-nonliteral", "-Wno-unused-variable", "-Wno-unused-function", "-Wno-sign-compare", "-Wno-error=implicit-function-declaration", "-Wno-int-conversion"]
```

Comment:
Are this strictly necessary? They're only disabling warnings - if glib has "warnings as errors" i would disable that one setting, rather than disable some specific ones here. The qnx compiler is actually gcc - I would expect the same warnings to appear when using gcc for any other platform

Reply:
Let's try without this flags and try to move this PR forward 😄 
```suggestion
```